src/main/resources/templates/probes/listing.html

```diff
@@ -2,25 +2,31 @@
 <html lang="en" data-layout-decorate="~{layouts/main}">
 <head>
     <title>Probes</title>
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/datatables/1.10.19/js/jquery.dataTables.min.js"></script>
+    <script src="" data-th-src="@{/js/index.js}" type="text/javascript"></script>
 </head>
 <body>
 
 <section data-layout-fragment="content">
     <h1>Probes</h1>
     <p>This is the list of probes currently implemented.</p>
-    <table class="table">
+    <table class="table" id="myTable">
```

Comment:
maybe something else than what is on the example.
```suggestion
    <table class="table" id="probes-table">
```